packages/boot/test/bootstrapTests/test-orchestration.ts

```diff
@@ -140,7 +140,7 @@ test.serial('stakeAtom - smart wallet', async t => {
     invitationSpec: {
       source: 'agoricContract',
       instancePath: ['stakeAtom'],
-      callPipe: [['makeCreateAccountInvitation']],
+      callPipe: [['makeNewAccountInvitation']],
```

Comment:
should be `makeAccountInvitation`?

Reply:
This was discussed [above](https://github.com/Agoric/agoric-sdk/pull/9312#discussion_r1586552809) - it's a function that makes an invitation to make an account, `makeMakeAccountInvitation` .

I don't feel strongly either way - `makeAccountInvitation` is potentially misleading, `makeNewAccountInvitation` could be too since we don't use "new" anywhere here.

Reply:
I think the historical pattern for this would be `makeAcountInvitationMaker`, which then would produce something you would call `makeAccountInvitation`